content/articles/what-happens-when-a-domain-expires.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: What happens when a domain expires?
+excerpt: Description of the different statuses a domain can have after its expiration.
+categories:
+- Domains
+---
+
+# What happens when a domain expires?
+
+When a domain expires, it enters into a grace period. Most registry offer a non-guaranteed 29-day grace period for renewal for many TLDs. Each registry has different rules for their grace periods. There is no guaranteed grace period. 
+
+After a domain has passed through any applicable expiration grace period, the domain may go into redemption status. "Redemption Grace Period" or RGP, is the grace period provided after the registrar has issued a delete request to the registry.
```

Comment:
"the domain may go into *a* redemption status. *A* "Redemption Grade Period", or RGP, is the grace period provided after the registrar has issued a delete request to the registry."